.tekton/create-ci-image-pipelinerun.yaml

```diff
@@ -0,0 +1,114 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: create-ci-image-pipelinerun
+  annotations:
+    pipelinesascode.tekton.dev/on-cel-expression: |
+      event == "pull_request" && "Dockerfile.CI".pathChanged() || event == "push" && "Dockerfile.CI".pathChanged()
+    pipelinesascode.tekton.dev/max-keep-runs: "5"
+spec:
+  params:
+    - name: repo_url
+      value: "{{ repo_url }}"
+    - name: revision
+      value: "{{ revision }}"
+  pipelineSpec:
+    params:
+      - name: repo_url
+      - name: revision
+    workspaces:
+      - name: source
+      - name: basic-auth
+    tasks:
+      - name: fetch-repository
+        taskRef:
+          kind: ClusterTask
+          name: git-clone
+        workspaces:
+          - name: output
+            workspace: source
+          - name: basic-auth
+            workspace: basic-auth
+        params:
+          - name: url
+            value: $(params.repo_url)
+          - name: revision
+            value: $(params.revision)
+      - name: generate-image-name
+        taskSpec: 
+          results:
+            - name: image-name
+            - name: image-name-timestamp
+          steps: 
+            - name: generate-image-name
+              image: quay.io/openshift-pipeline/ci
+              script: | 
+                #!/usr/bin/env bash
+                BRANCH_NAME={{ target_branch }}
+                IMAGE_NAME=""
+                IMAGE_NAME_TIMESTAMP=""
+                TIMESTAMP=$(date +"%y%m%d")
+
+                if [ "$BRANCH_NAME" == "master" ]; then 
+                  if [ {{ event_type }} == "pull_request" ]; then
+                      IMAGE_NAME="image-registry.openshift-image-registry.svc:5000/pipelines-ci/ci:latest"
+                      IMAGE_NAME_TIMESTAMP="image-registry.openshift-image-registry.svc:5000/pipelines-ci/ci:5.0"-"$TIMESTAMP"
```

Comment:
You don't need to end and start quotes `"...ci:5.0"-"$TIMESTAMP"`, simply use the variable inside the string